webapp_frontend/public/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="h-100">
```

Comment:
Can you really put styles on this element?

Reply:
Yes, works totally fine